serialzy/serializers/catboost.py

```diff
@@ -1,14 +1,26 @@
+from os import PathLike
+from pathlib import Path
 from typing import BinaryIO, Callable, Type, Union, Any, Optional
 
 from serialzy.errors import SerialzyError
-from serialzy.serializers.base_model import ModelBaseSerializer, serialize_to_file, deserialize_from_file
+from serialzy.serializers.base_model import (
+    ModelBaseSerializer,
+    serialize_to_file,
+    deserialize_from_file,
+    unpack_model_file
+)
 
 
 # noinspection PyPackageRequirements
 class CatboostPoolSerializer(ModelBaseSerializer):
     def __init__(self):
         super().__init__("catboost", __name__)
 
+    def unpack_model(self, source: BinaryIO, dest_dir: Union[str, PathLike]) -> PathLike:
+        model_path = Path(dest_dir) / "model.bin"
```

Comment:
This code is repeating at all serializers.
Maybe you should make an additional method in ModelBaseSerializer like:
```
def _unpack_model(self, source, dest_dir, dest_file, is_archive):
    ...
```
and use it like
```
def unpack_model(...):
    return self._unpack_model(source, dest_dir, 'model.bin', is_archive=False)
```